tests/text/test_rouge.py

```diff
@@ -11,189 +11,179 @@
 
-PRECISION = 0
-RECALL = 1
-F_MEASURE = 2
-
 SINGLE_SENTENCE_EXAMPLE_PREDS = "The quick brown fox jumps over the lazy dog"
 SINGLE_SENTENCE_EXAMPLE_TARGET = "The quick brown dog jumps on the log."
 
+SINGLE_SENTENCE_EXAMPLE_RESULTS = {
+    "rouge1_precision": torch.tensor(0.6666666666666666),
+    "rouge1_recall": torch.tensor(0.75),
+    "rouge1_fmeasure": torch.tensor(0.7058823529411765),
+    "rouge2_precision": torch.tensor(0.25),
+    "rouge2_recall": torch.tensor(0.2857142857142857),
+    "rouge2_fmeasure": torch.tensor(0.26666666666666666),
+    "rougeL_precision": torch.tensor(0.5555555555555556),
+    "rougeL_recall": torch.tensor(0.625),
+    "rougeL_fmeasure": torch.tensor(0.5882352941176471),
+    "rougeLsum_precision": torch.tensor(0.5555555555555556),
+    "rougeLsum_recall": torch.tensor(0.625),
+    "rougeLsum_fmeasure": torch.tensor(0.5882352941176471),
+}
+
 PREDS = "My name is John".split()
 TARGETS = "Is your name John".split()
 
-BATCHES_RS_PREDS = [SINGLE_SENTENCE_EXAMPLE_PREDS]
-BATCHES_RS_PREDS.extend(PREDS)
-BATCHES_RS_TARGETS = [SINGLE_SENTENCE_EXAMPLE_TARGET]
-BATCHES_RS_TARGETS.extend(TARGETS)
+PREDS_SPLIT_RESULTS = {
+    "rouge1_precision": torch.tensor(0.25),
+    "rouge1_recall": torch.tensor(0.25),
+    "rouge1_fmeasure": torch.tensor(0.25),
+    "rouge2_precision": torch.tensor(0.0),
+    "rouge2_recall": torch.tensor(0.0),
+    "rouge2_fmeasure": torch.tensor(0.0),
+    "rougeL_precision": torch.tensor(0.25),
+    "rougeL_recall": torch.tensor(0.25),
+    "rougeL_fmeasure": torch.tensor(0.25),
+    "rougeLsum_precision": torch.tensor(0.25),
+    "rougeLsum_recall": torch.tensor(0.25),
+    "rougeLsum_fmeasure": torch.tensor(0.25),
+}
 
 BATCHES = [
     dict(preds=[SINGLE_SENTENCE_EXAMPLE_PREDS], targets=[SINGLE_SENTENCE_EXAMPLE_TARGET]),
     dict(preds=PREDS, targets=TARGETS),
 ]
 
-
-def _compute_rouge_score(preds: List[str], targets: List[str], use_stemmer: bool):
-    scorer = RougeScorer(ROUGE_KEYS, use_stemmer=use_stemmer)
-    aggregator = BootstrapAggregator()
-    for pred, target in zip(preds, targets):
-        aggregator.add_scores(scorer.score(pred, target))
-    return aggregator.aggregate()
-
-
-@pytest.mark.skipif(not (_NLTK_AVAILABLE or _ROUGE_SCORE_AVAILABLE), reason="test requires nltk and rouge-score")
+BATCHES_RESULTS = {
+    "rouge1_precision": torch.tensor(0.3333333333333333),
+    "rouge1_recall": torch.tensor(0.35),
+    "rouge1_fmeasure": torch.tensor(0.3411764705882353),
+    "rouge2_precision": torch.tensor(0.05),
+    "rouge2_recall": torch.tensor(0.05714285714285714),
+    "rouge2_fmeasure": torch.tensor(0.05333333333333333),
+    "rougeL_precision": torch.tensor(0.3111111111111111),
+    "rougeL_recall": torch.tensor(0.325),
+    "rougeL_fmeasure": torch.tensor(0.31764705882352945),
+    "rougeLsum_precision": torch.tensor(0.3111111111111111),
+    "rougeLsum_recall": torch.tensor(0.325),
+    "rougeLsum_fmeasure": torch.tensor(0.31764705882352945),
+}
```

Comment:
Instead of hardcoding the results, could you maybe compare to the other rouge-score package?

Reply:
Maybe, can I use the referred `rouge-score` package and add this one into the tests' requirements only?

Reply:
exactly. That would be perfect, so that we also have tests if we decide to change the examples for tests.